### PR TITLE
[16.0][FIX] website_cookiebot: disable for sure

### PR DESCRIPTION
**Pull request by chienandalu**
If we don't remove the key the assets will keep trying to load, which isn't what we'd expect.

cc @Tecnativa TT50520

please review @pedrobaeza @CarlosRoca13

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Erasing the identifier seems a bit aggressive. Can we convert the boolean into a regular field and change the template to not load it if not checked?

**Comment by chienandalu** [comment]
We could do it but that will require more changes and a migration script

**Comment by pedrobaeza** [comment]
Well, I think that the way to go. If not, you lose the ID if you want to disable it temporarily or make some tests.

**Comment by chienandalu** [comment]
Ok, done :)

File: website_cookiebot/wizards/res_config_settings.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class ResConfigSettings(models.TransientModel):
     )
     cookiebot_enabled = fields.Boolean(
         string="Cookiebot",
+        related="website_id.cookiebot_enabled",
         compute="_compute_cookiebot_enabled",
```
Comment (pedrobaeza) [inline website_cookiebot/wizards/res_config_settings.py]: Remove this attribute

Comment (chienandalu) [reply website_cookiebot/wizards/res_config_settings.py]: It's used as an onchange to set it off when the Odoo's own cookies setting is on

Comment (pedrobaeza) [reply website_cookiebot/wizards/res_config_settings.py]: Then just remove the related and put an inverse for clearer understanding.

Comment (chienandalu) [reply website_cookiebot/wizards/res_config_settings.py]: Sorry, I missread it. It was useless...

**Review by CarlosRoca13 — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**